GillianCore/engine/GeneralSemantics/General/GInterpreter.ml

```diff
@@ -366,14 +363,11 @@ struct
                  ([ EProc (Val.from_literal (String pid)) ], state))
       in
       let caller = CallStack.get_cur_proc_id cs in
-      let () =
-        if not (is_internal_proc prog caller || is_internal_proc prog pid) then
-          CallGraph.add_proc_call call_graph caller pid
-      in
+      let () = CallGraph.add_proc_call call_graph caller pid in
```

Comment:
We need them for the case when an internal procedure calls a non-internal procedure, which I was told can happen (especially in JavaScript). It would be for all analyses.